packages/notifications-library/tests/conftest.py

```diff
@@ -76,8 +76,12 @@ def external_environment(request: pytest.FixtureRequest) -> EnvVarsDict:
     """
     envs = {}
     if envfile := request.config.getoption("--external-envfile"):
-        assert isinstance(envfile, Path)
         print("🚨 EXTERNAL `envfile` option detected. Loading", envfile, "...")
+
+        assert isinstance(envfile, Path)
+        assert envfile.exists()
+        assert envfile.is_file()
```

Comment:
I assume the second assert would be enough to check it also exists.

Reply:
@bisgaard-itis True. Are you a mathematitian? ;-)